### ISSUE TITLE
googkit installed by pip does not work on Windows

### ISSUE THREAD

**Issue by Kuniwak** [googkit/googkit#36]
googkit did not work on Windows7.
And googkit was installed by pip.

```
C:\Users\###\Documents\test>python --version
Python 3.3.2

C:\Users\###\Documents\test>googkit init
[Error] Template directory is not found: C:\Python33\lib\site-packages\etc\template
```

Actual template directory path is `C:\Python33\etc\template`,
and actual googkit root is `C:\Python33\Lib\site-packages\googkit`

**Comment by Kuniwak** [comment]
But googkit installed by `python setup.py install` works.
Then, template directory path is:

```
C:\Python33\Lib\site-packages\googkit-0.1.0-py3.3.egg\etc\template
```

**Comment by cocopon** [comment]
Oh... I have to buy Windows to solve this problem...

**Comment by Kuniwak** [comment]
I suggest to add the following section to README.

```
Dear Windows user,

You can't use googkit installed by `pip install gogokit`.
Instead, try following procedure.

1. Download googkit
2. Open googkit directory by Command Prompt
3. `python setup.py install`
```

**Comment by Kuniwak** [comment]
I found the related question: [setuptools: package data folder location](http://stackoverflow.com/questions/4519127/setuptools-package-data-folder-location/5423147#5423147)

**Comment by Kuniwak** [comment]
It does seems that we have 3 ways.
1. Unsupport googkit installed by `pip install googkit` on Windows (it seems better)
2. Switch the `etc` directory to `package_data` from `data_files`
3. Check the path `os.path.join(sys.prefix, 'template')` in `lib.path.template()` (it seems not good)

**Comment by cocopon** [comment]
Oh, I reproduced it in OS X. It's a problem specific to `pip`.

Project root:  `/Library/Python/2.7/site-packages/googkit`
Data directory: `/System/Library/Frameworks/Python.framework/Versions/2.7/etc`

I'll try to resolve it... (I think no.2 is better idea)

**Comment by cocopon** [comment]
No.2 seems to require putting `__init__.py` into each data directory.
It's not smart...

http://stackoverflow.com/questions/13307408/python-packaging-data-files-are-put-properly-in-tar-gz-file-but-are-not-install

**Comment by cocopon** [comment]
Hmm...really needs `__init__.py`? I'll try it later:
http://docs.python.jp/2/distutils/setupscript.html

**Comment by Kuniwak** [comment]
I think that only `completion` directory should be specified `data_files`.

**Comment by cocopon** [comment]
I fixed the issue in my environment via (7dd8ab9da24bd40822da9b74f3703b872c4512c8). How about you?

**Comment by Kuniwak** [comment]
Fixed (d2837cd).
